Add udev rule for WiFi devices

WiFi power save settings are not always applied at boot time. When this occurs
the kernel initialization of the WiFi interface has not been completed when
"laptop-mode force" runs at boot. That this issue is seen now is probably
a side effect of CL 4683f33 ("laptop-mode-tools: Refactor
usb_autosuspend::listed_by_id") which reduces the runtime of "laptop-mode force"
significantly.

Add a udev rule for WiFi/net devices to make sure laptop-mode can apply the
configured power save settings.

From: Matthias Kaehlcke <mka@chromium.org>
Project-Bug-URL: https://crbug.com/666459

--- laptop-mode-tools-1.59/etc/rules/99-laptop-mode.rules
+++ laptop-mode-tools-1.59/etc/rules/99-laptop-mode.rules
@@ -1,3 +1,4 @@
 ACTION=="change", SUBSYSTEM=="power_supply", RUN+="/lib/udev/lmt-udev auto"
 ACTION=="add|remove", SUBSYSTEM=="machinecheck", RUN+="/lib/udev/lmt-udev auto"
 ACTION=="add", SUBSYSTEM=="usb", RUN+="/lib/udev/lmt-udev force modules=usb-autosuspend devices=%k"
+ACTION=="add", SUBSYSTEM=="net", RUN+="/lib/udev/lmt-udev force modules=wifi-nl80211-powermgmt"
